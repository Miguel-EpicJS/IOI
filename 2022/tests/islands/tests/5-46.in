4e5150f5a11b21d9ac3b44f2259e7a3452343ad4
8 15
4 6
1 2
0 4
0 1
0 6
6 3
0 3
1 7
5 7
5 6
1 6
6 3
4 6
7 1
4 5
