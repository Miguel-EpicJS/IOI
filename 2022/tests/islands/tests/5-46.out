91041670623a69783795ed82bda33426101ce311
OK
1
12
2 14 8 13 7 8 14 2 3 13 7 3
